src/triage/component/catwalk/evaluation.py

```diff
@@ -209,12 +279,20 @@ def _evaluations_for_threshold(
                 raise metrics.UnknownMetricError()
 
             for parameter_combination in parameters:
-                value = self.available_metrics[metric](
-                    predictions_proba,
-                    predicted_classes_with_labels,
-                    present_labels,
-                    parameter_combination,
-                )
+                try:
+                    value = self.available_metrics[metric](
+                        predictions_proba,
+                        predicted_classes_with_labels,
+                        present_labels,
+                        parameter_combination,
+                    )
+                except:
```

Comment:
is this really *any* exception? or is it reliably a `KeyError` or something?

catching *all* possible exceptions can obscure actual bugs, and so it's best to avoid, when possible.

Reply:
Probably not _any_ exception, but likely a list of exceptions. I can run through the metrics and grab out the appropriate exceptions.